tests/test_auth.py

```diff
@@ -139,13 +148,21 @@ def test_successful_authentication_new_user(self, token_mock, request_mock, algo
     def test_authenticate_no_code_no_state(self):
         """Test authenticate with wrong parameters."""
 
-        self.assertEqual(self.backend.authenticate(code='', state=''), None)
+        # there are no GET params
+        request = RequestFactory().get('/foo')
+        request.session = {}
+        with self.assertRaises(SuspiciousOperation) as context:
```

Comment:
Instead of checking the message in a different assertion you can always use:
self.assertRaisesMessage(exception, message)